src/config/ci-sim.toml

```diff
@@ -7,3 +7,7 @@ max_concurrent_creating_streaming_jobs = 0
 
 [meta]
 meta_leader_lease_secs = 10
+
+[meta.developer]
+meta_actor_cnt_per_worker_parallelism_soft_limit = 65536
+meta_actor_cnt_per_worker_parallelism_hard_limit = 65536
```

Comment:
How's this related with etcd vs sql? :eyes:

Reply:
Let me add this to the list of things to follow up on.